distribution/ddf-common/src/main/resources-filtered/etc/custom.system.properties

```diff
@@ -183,6 +183,12 @@ org.codice.ddf.platform.util.http.initialRetryInterval=10
 javax.xml.parsers.DocumentBuilderFactory=org.apache.xerces.jaxp.DocumentBuilderFactoryImpl
 javax.xml.transform.TransformerFactory=org.apache.xalan.processor.TransformerFactoryImpl
 
+#
+# Metadata Size Limit
+#
+
+# Sets the size limit of the metadata within the metacards. Size is in megabytes. Default is 5mb.
```

Comment:
❓ Why limit this to megabytes?  Any reason not to make this bytes?  That would give more control and allow us to remove the conversion code.  I would not think bytes would be a problem since this is an advance option.

Reply:
Makes sense. Fixed